.github/workflows/helm_charts_scalar.yml

```diff
@@ -177,6 +177,7 @@ jobs:
           kubectl create secret generic auditor-keys --from-file=certificate=.github/auditor-cert.pem --from-file=private-key=.github/auditor-key.pem
           kubectl create -f .github/cassandra.yaml
           echo "Wait until cassandra is running" && sleep 100
+          kubectl create -f .github/schema-loading.yaml
```

Comment:
I recommend that waiting for the job execution to complete makes CI more stable.

```
kubectl wait --for=condition=complete --timeout=60s job/schema-loading
```

Reply:
It is also better to wait for Pods to be ready for Cassandra as well, as CI is expected to be more stable and finish sooner (no need to wait for 100s).

Reply:
@superbrothers  
Thank you for your comment!
I agree with waiting for Cassandra in another way.

@ymorimo  

I think the `kubectl wait` command is not appropriate as a way to wait for Cassandra to start.
This is because Cassandra is not ready for accepting requests in the container even if the pod (container) status is `Running`.
So, I usually use the following command to wait for Cassandra to accept the requests, in my local environment testing.
```console
echo -n "Wait for Cassandra* "
until kubectl exec cassandra-0 -- cqlsh 2> /dev/null; do  sleep 0.5; echo -n "#"; done
```
This command checks whether Cassandra is ready to accept a request or not using `cqlsh`.
I think we can use the same command in this CI for waiting for Cassandra to start.

Reply:
It looks like you can use `kubectl wait` command by setting up the readiness probe for cassandra Pod.

```
diff --git a/.github/cassandra.yaml b/.github/cassandra.yaml
index e0a8a67..fc0e677 100644
--- a/.github/cassandra.yaml
+++ b/.github/cassandra.yaml
@@ -21,6 +21,9 @@ spec:
           env:
             - name: CASSANDRA_SEEDS
               value: "cassandra-0.cassandra-headless.default.svc.cluster.local"
+          readinessProbe:
+            exec:
+              command: ["cqlsh", "-e", "SELECT now() FROM system.local;"]
 ---
 apiVersion: v1
 kind: Service
```

```
kubectl wait --for=condition=Ready pod/cassandra-0
```

However, this change should be implemented in a separate PR.

Reply:
@superbrothers  
Thank you for your suggestion!
We will update it in another PR.